src/compiler/types.ts

```diff
@@ -3522,9 +3532,10 @@ namespace ts {
     }
 
     export const enum InferencePriority {

```

Comment:
should probably be `@internal`

Reply:
Probably, but not really related to this PR since the type already existed before. I'm going to leave it be for now.